### PR TITLE
frontend: make locize-pull

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: frontends/web/src/locales/en/app.json [hunk 1]
```diff
@@ -1030,27 +1030,27 @@
...
-      "title": "What is a Passphrase?"
+      "button": "Learn how this works",
+      "message": "A wallet is created (derived) from a very big random number, also known as a seed. This seed is created when you first set up your BitBox02 and is backed up with the microSD card or 24 words. Anyone who has access to the seed has full control over the funds on that wallet.\n\nA passphrase is an <strong>optional secret</strong>, added to the seed. When using a passphrase, each passphrase creates a new wallet based on the seed + passphrase (optional secret). A passphrase can be anything: letters, words, special characters or it can even be empty. The default wallet is in fact derived from the seed + empty passphrase.",
+      "title": "What is a passphrase?"
     },
     "why": {
-      "button": "Why use a passphrase?",
-      "message": "The BitBox02 protects the seed against extraction, but the backup (microSD card or 24 words) gives full access to your wallet. That is why it should be stored in a secure location!\n\nWhen using a passphrase, each passphrase unlocks a new wallet. A different passphrase will create a different wallet.",
+      "button": "Why use a passphrase",
```
Comment (benma) [inline frontends/web/src/locales/en/app.json:1052]: why no question mark?

Comment (thisconnect) [reply frontends/web/src/locales/en/app.json:1052]: yes, this is the button text, the actual screen title has a question mark

**Comment by thisconnect** [comment]
LGTM

**Comment by thisconnect** [comment]
Thanks for the PR I'll update the text again today.

**Comment by thisconnect** [comment]
Related https://github.com/digitalbitbox/bitbox-wallet-app/pull/1459

**Pull request closed without merging**